site/en/userGuide/schema/sparse_vector.md

```diff
@@ -261,7 +261,7 @@ const indexParams = await client.createIndex({
     metric_type: MetricType.IP,
     index_type: IndexType.SPARSE_INVERTED_INDEX,
     params: {
-      inverted_index_algo: 'DAAT_MAXSCORE',
+      inverted_index_algo: 'DAAT_MAXSCORE', // Algorithm used for building and querying the index
```

Comment:
it's better to show the candidate values.
Also need to check the available version, I will check this.